test/unit/lodash.spec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+
+            var params, number, lower, upper;
+
+            before(function ()
+            {
+                params = datasets.clamp();
+                number = params[0];
+                lower = params[1];
+                upper = params[2];
+            });
+
+            it('should verify params', function ()
+            {
+                expect(params).to.have.length(3);
```

Comment:
In fact we don't need to check number of params, because tested method always should take 3 of them.